OpenCL.pc.in

```diff
@@ -0,0 +1,8 @@
+libdir=@CMAKE_INSTALL_FULL_LIBDIR@
+includedir=@CMAKE_INSTALL_FULL_INCLUDEDIR@
+
+Name: OpenCL
+Description: Khronos OpenCL ICD Loader
+Version: 3.0
```

Comment:
Confirming: It's OK if this version number is different than our SOVERSION?

If so, this is another place that would be great to automate with #182!

Reply:
> Confirming: It's OK if this version number is different than our SOVERSION?

This mimics what `ocl-icd` does here: https://github.com/OCL-dev/ocl-icd/blob/master/OpenCL.pc.in where @OPENCL_VERSION@ is currently evaluated to `3.0`

> If so, this is another place that would be great to automate with https://github.com/KhronosGroup/OpenCL-ICD-Loader/pull/182!

Indeed